libs/types/src/ui_theme.ts

```diff
@@ -1,5 +1,5 @@
 /** Supported screen types for VxSuite apps. */
-export type ScreenType = 'builtIn' | 'elo13' | 'elo15';
+export type ScreenType = 'builtIn' | 'elo13' | 'elo15' | 'lenovo';
```

Comment:
Might be worth making this `lenovo-thinkpad` or `lenovo-thinkpad-15` or even something more specific, but I can live with this too.

Reply:
Yeah, that's a good call - will go with `lenovo-thinkpad-15` for clarity